frontend/src/components/common/presentational/profilePageHeader/ProfilePageHeader.tsx

```diff
@@ -1,21 +1,28 @@
-import profileIcon from "@src/assets/svgs/Ellipse20Pic.svg";
 import emailIcon from "@src/assets/svgs/Email_Frame.svg";
 import phoneIcon from "@src/assets/svgs/Phone.svg";
+import { useGetUserQuery } from "@src/store/reducers/employees-api";
 import "@src/components/common/presentational/profilePageHeader/profilePageHeader.scss";
 
 function ProfilePageHeader() {
+  const { data: userData } = useGetUserQuery();
   return (
     <>
       <div className="profile-Header-main">
         <div className="icon-Container">
-          <img className="profile-pic" src={profileIcon} alt="profile pic" />
+          <img
+            className="profile-pic"
+            src={userData?.avatar}
+            alt="profile pic"
+          />
         </div>
 
         <div className="employee-Header-Div-Two">
           <div className="subContainer-row-one">
             <div className="name">
-              {/* {employeeData?.user?.first_name} {employeeData?.user?.last_name} */}
-              Akita
+              {" "}
+              {userData?.first_name && userData?.last_name
+                ? `{userData?.first_name} {userData?.last_name}`
+                : "Name"}{" "}
```

Comment:
Don't add this ternary op. Just show firstname and lastname. If these are empty, let it be empty.